include/util/static_rtree.hpp

```diff
@@ -43,16 +43,19 @@ namespace util
 template <class EdgeDataT,
           class CoordinateListT = std::vector<Coordinate>,
           bool UseSharedMemory = false,
-          std::uint32_t BRANCHING_FACTOR = 128,
-          std::uint32_t LEAF_NODE_SIZE = 128,
-          std::uint32_t MAX_NUM_CANDIDATES = LEAF_NODE_SIZE * 10>
+          std::uint32_t BRANCHING_FACTOR = 64,
```

Comment:
That would be awesome. :+1: Last benchmarks I did were only spot checks and that was before some bigger structural changes in the RTree so we might see different results.